wg-qa/README.md

```diff
@@ -0,0 +1,28 @@
+# QA Working Group
+
+Oversees stability, quality, and testing criteria within Electron.
+
+## Membership
+
+| Avatar| Name | Role | Time Zone |
+| ----------------- | ----------------- | ----------------- | ----------------- |
+| <img src="https://github.com/MariaBeliaeva.png" width=100 alt="@MariaBeliaeva">  | Maria Beliaeva [@MariaBeliaeva](https://github.com/MariaBeliaeva)| **Chair** | PT (San Francisco) |
+| <img src="https://github.com/VerteDinde.png" width=100 alt="@VerteDinde">  | Keeley Hammond [@VerteDinde](https://github.com/VerteDinde) | Member | PT (Portland) |
+| <img src="https://github.com/cruzerld.png" width=100 alt="@cruzerld">  | David Cruz [@cruzerld](https://github.com/cruzerld) | Member | ET (New York City) |
+| <img src="https://github.com/alexeykuzmin.png" width=100 alt="@alexeykuzmin">  | Alexey Kuzmin [@alexeykuzmin](https://github.com/alexeykuzmin) | Member | CET (Prague) |
+| <img src="https://github.com/sofianguy.png" width=100 alt="@sofianguy"> | Sofia Nguy [@sofianguy](https://github.com/sofianguy) | Member | PT (San Francisco) |
+
+
+
+## Parent Group
+
+* Releases WG
+
+## Areas of Responsibility
+
+* Strongly advises the stable release.
+* Long-term would oversee test automation and greenlight the stable release.
```

Comment:
Is the AFP going to live under this WG?

Reply:
I recall some discussion at Summit about AFP falling under QA WG, but it also make senses under Releases WG. However, QA is under Releases so maybe? 🤔